test/src/test/java/org/corfudb/runtime/view/StreamViewTest.java

```diff
@@ -480,17 +483,19 @@ private void synchronousPrefixTrim(CorfuRuntime runtime, long epoch, long addres
      */
     @Test
     public void txLogTrim() {
+        final String streamTag = "tx_stream_tag";
+        final UUID tagId = CorfuRuntime.getStreamID(streamTag);
```

Comment:
same here, what is this fake tagId representing?

Reply:
same as before.. using the tag to mimic the behavior of the transaction stream.  The tag is being used when we try to get the streamView in L516.